publisher/publisher.go

```diff
@@ -48,6 +56,22 @@ func init() {
 	}
 
 	formatFileName = formatFile.Name()
+
+	// Set up the log format file for parsing in the future.
```

Comment:
If you pull `logFormatV2` into `logFormat` like I mentioned above all of this extra code can be avoided, since there's already existing code to write that format file to the FS.